rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java

```diff
@@ -140,7 +140,7 @@ void addSignature_fedPubKey_belongs_to_active_federation() throws Exception {
             createHash3(1).getBytes()
         );
 
-        verify(provider, times(1)).getPegoutsWaitingForSignatures();
+        verify(provider, times(0)).getPegoutsWaitingForSignatures();
```

Comment:
```suggestion
        verify(provider, never()).getPegoutsWaitingForSignatures();
```

Reply:
Just for my knowledge, why is this never occurring now?

Reply:
because we are returning early when null / empty signatures :)

Reply:
thanks :)